.github/workflows/ci-cd.yml

```diff
@@ -1377,12 +1377,13 @@ jobs:
           matrix.target-container.tag
         }}
 
-    continue-on-error: >-
-      ${{
-        contains(matrix.target-container.tag, 'ubi9')
-        && true
-        || false
-      }}
+    continue-on-error: true
```

Comment:
With this, failures won't contribute to the gating. Make sure to re-enable the important ones before merging.